database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -188,6 +188,13 @@ abstract class SlickDatabase(override val originalDatabaseConfig: Config) extend
     }
   }
 
+  protected[this] def isOfficiallySupportedDatabase(): Boolean = {
```

Comment:
I would make this a little more specific, eg maybe `databaseTriggeredSummarizationQueue` since:
1. It's not used for any other switching at the moment
2. We might choose to start officially supporting a non-triggered DBMS some day
3. We might choose to stop officially supporting one of these DBMSs some day